<launch>
 <arg name="server" default="192.168.1.230"/>
 <node pkg = "vrpn_client_ros" name = "vrpn_client_node" type="vrpn_client_node">
 <rosparam subst_value="true">
 server: $(arg server)
 port: 3883
 update_frequency: 100.0
 frame_id: world
 # Use the VRPN server's time, or the client's ROS time.
 use_server_time: false
 broadcast_tf: true
 # Must either specify refresh frequency > 0.0, or a list of trackers to create
 refresh_tracker_frequency: 1.0
 #trackers:
 #- FirstTracker
 #- SecondTracker
 </rosparam>
 </node>
</launch>
